cisagov/ScubaGear
# Revert run OPA tests action from <0.50 to latest

#  #

## 🗣 Description ##

- In `run_opa_tests.yml` changed OPA version from <0.50 to latest
- In `OPA.ps1` bumped the expected version from v0.42.1 to v0.59.0
 



## 💭 Motivation and context ##

During the 0.30 release the OPA version had to be switched from latest to <0.50, #193. PR #195 has been merged into Flipper which clears up the Rego errors preventing test actions from updating.  

Resolves #196
Resolves #593  






## 🧪 Testing ##

Download OPA v0.59.0 from https://www.openpolicyagent.org/docs/latest/. 
- Run `./opa_windows_amd64.exe check Rego Testing/Unit/Rego -- strict`
- Run `Invoke-Scuba` cmdlet against all products for both gcc/gcchigh







## ✅ Pre-approval checklist ##






      in code comments.
      to reflect the changes in this PR.
- [x] Rebased with most up to date changes from Flipper release

## ✅ Pre-merge checklist ##





- [x] Ensure any relevant issues are referenced in the PR description
- [x] Run tests locally
